engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -306,7 +306,16 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
       validatedNamespace <- buildValidatedNamespace(factory, sourceAndResolvers._1, sourceAndResolvers._2)
       _ = pushNamespaceMetadata(validatedNamespace)
       ewd <- fromEither[IO](buildWorkflowDescriptor(id, validatedNamespace, workflowOptions, labels, conf, pathBuilders, outputRuntimeExtractor).toEither)
-    } yield ewd
+    } yield {
+
+      // Debugging #4117 - make sure we are creating an importable directory out of the zip (AEN 2018-11-14)
+      zippedImportResolver match {
+        case Some(zir) => workflowLogger.info(s"Using zip import resolver '${zir.name}'")
```

Comment:
Is it possible to log either here (or at unzip time) how many items were actually unzipped? One hypothesis might be "the unzip somehow incorrectly contained zero (or wrong) files"?

Reply:
not sure we want to log that kind of detail in all cases...